web/frontend/src/index.tsx

```diff
@@ -224,38 +224,42 @@ const AppContainer = () => {
     };
 
     async function fetchData() {
+      const response = await fetch(ENDPOINT_PERSONAL_INFO, req);
+      let result;
       try {
-        const res = await fetch(ENDPOINT_PERSONAL_INFO, req);
-
-        if (res.status !== 200) {
-          const txt = await res.text();
-          throw new Error(`unexpected status: ${res.status} - ${txt}`);
+        if (response.status === 200) {
+          result = await response.json();
+        } else if (response.status === 401) {
+          result = {
+            isLoggedIn: false,
+            firstName: '',
+            lastName: '',
+            authorization: {},
+          };
+        } else {
+          const txt = await response.text();
```

Comment:
if the response did not return a 200, doesn't this always throw ?

Reply:
if it's not 200 or 401

Reply:
what I meant is: isn't `response.text()` always going to throw before your `throw new Error()`, since we're not in a 200 ?

Reply:
@pierluca It succeeds. I tested it. 

My test: I restarted the backend, and refreshed the UI causing the backend to return 504. the frontend then displayed the intended message.